code/API_definitions/Get DeviceLocation.yaml

```diff
@@ -0,0 +1,280 @@
+openapi: 3.0.3
+info:
+  title: Retrieve Device location API
+  description: Service Enabling Network Function API for location retrieval from a device identifier.
+  termsOfService: http://swagger.io/terms/
+  contact:
+    email: project-email@sample.com
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0-wip
+externalDocs:
+  description: Product documentation at Camara
+  url: https://github.com/camaraproject/
+security:
+  - oAuth2ClientCredentials: []
+#  - BasicAuth: []
+#  - apiKey: []
+  - three_legged:
+    - device-location-provide
+servers:
+  - url: '{apiRoot}/{basePath}'
+    variables:
+      apiRoot:
+        default: http://localhost:9091
+        description: API root
+      basePath:
+        default: location/v0
+        description: Base path for the device location API
+paths:
+  /retrieve-location:

```

Comment:
I have to note here, that our guidelines says: "Verbs use is not allowed."
Maybe: location-detection or location-retrieval

Reply:
@jlurien ok, then it's fine for me ( in the location API there was also a 'verify' initially )